src/cmd/servo.rs

```diff
@@ -138,7 +142,20 @@ pub fn run_list(args: &ArgsList) -> Result<()> {
         println!("{}", keys.join(" "));
         return Ok(());
     }
-    println!("{}", list);
+    if args.json {
+        println!("{}", list);
+        return Ok(());
+    }
+    println!("product         serial                          usb_sysfs_path");
```

Comment:
isn't this use same format (offset values) as below code?
```
println!("{:16}{:24}\t{}", "product", "serial", "usb_sysfs_path");
```

Reply:
Clippy complains if I use the format string with static strings... that's why I chose this way...

Reply:
true, although it might become more readable/maintainable with what clippy suggests:
```
println!("{:16}{:24}\tusb_sysfs_path", "product", "serial");
```

Reply:
Hmm, I think clippy missed the point, if we change some items to display, or change the order, that format can not simply swap it. Using the same format string is the key point, so that we can change the data easily.

Reply:
OK, anyway it seems hard to escape from the clippy optimization. So after introducing some better formatting (for showing table) we will revisit it.